Client/Frontend/Widgets/CurveView.swift

```diff
@@ -0,0 +1,63 @@
+/* Code for drawing the urlbar curve */
```

Comment:
When I started to do this, I created a separate URLBarView implementation for landscape, and needed to reuse this on it. I pulled it out for reuse. (I still kinda wish I had those separate implementations, but the view/constraint lifecycle on iOS doesn't make it trivial to trade out views on rotation). I don't have super strong opinions, but I don't think the encapsulation granted here really gains us anything so I didn't put it back. i.e. we don't gain anything by saying "No one except the urlbar should ever use this widget for anything!" (as opposed to, "no one outside should know about or change this variable/method")